### PR TITLE
[KOGITO-1257] Cucumber tests - install only needed dependent operators

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

https://issues.redhat.com/browse/KOGITO-1257

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by ricardozanini** [comment]
Please don't forget to check the requirements.

**Comment by sutaakar** [comment]
Thanks for reminding, I guess I really forgot

File: test/features/cli/cli_deploy_quarkus_service.feature [hunk 1]
```diff
@@ -27,7 +27,7 @@ Feature: CLI: Deploy quarkus service
 
   @persistence
   Scenario Outline: CLI deploy jbpm-quarkus-example service with persistence
-    Given Kogito Operator is deployed with dependencies
+    Given Kogito Operator is deployed with Infinispan
```
[the remark this answers is not in the document]
Comment (sutaakar) [reply test/features/cli/cli_deploy_quarkus_service.feature]: good point, will adjust the PR

[the remark this answers is not in the document]
Comment (sutaakar) [reply test/features/cli/cli_deploy_quarkus_service.feature]: @radtriste PR updated

**Pull request merged**